backend/utils/random.go

```diff
@@ -6,16 +6,13 @@ import (
 )
 
 // GenerateRandomSecureString generates a random secure string
-func GenerateRandomSecureString(length int) (string, error) {
+func GenerateRandomSecureString(length int) string {
```

Comment:
FYI: https://github.com/samber/lo#randomstring since we are already using `lo` in lots of places